src/world/Territory/Cell.cpp

```diff
@@ -11,10 +11,10 @@
 Sapphire::Cell::Cell() :
   m_bActive( false ),
   m_bLoaded( false ),
-  m_playerCount( 0 ),
-  m_bUnloadPending( false )
+  m_bUnloadPending( false ),
+  m_playerCount( 0 )
 {
-  m_bForcedActive = false;
+    m_bForcedActive = false;
```

Comment:
This goes for pretty much everywhere else too, but please configure whatever you're using to respect `.editorconfig` or see the style guide